### ISSUE TITLE
1.0.4 returning wrong value

### ISSUE THREAD

**Issue by mikilukasik** [mobz/get-timezone-offset#4]
![image](https://user-images.githubusercontent.com/12686567/93355858-6aaf0980-f836-11ea-9bbb-e135e909fa08.png)

**Comment by calvinckho** [comment]
@mobz Thanks for this wonderful package. I have the same issue of returning wrong offsets after upgrading to 1.0.4.

**Comment by fedyk** [comment]
Hey, the problem can be related to `nodejs` you are using. Please check if you have >= 12

**Comment by spencerwilson** [comment]
Agreed that `-60` is the expected output for this example. I confirmed that 1.0.4 returns the correct result in Node.js 12.21.0 [on RunKit](https://runkit.com/embed/4rnbh42z2dza)—and in fact 1.0.3 returns `undefined` in that same Node.js version! And same story on both counts for Node.js 13.14.0.

1.0.4 in Node.js 14.16.0+ returns `-60`. 1.0.3 in the same returns `-1500`.

TL;DR
- Use 1.0.3 if you're < Node 12
- Use 1.0.4 if you're >= Node 12

Node 10 is EOL on 2021-04-30 anyway ([source](https://nodejs.org/en/about/releases/)), so there are multiple reasons to prioritize upgrading.

**Comment by mobz** [comment]
Fixed in v1.0.5

this version works in all versions of node
